warehouse/assemble/datawave/pom.xml

```diff
@@ -475,33 +459,61 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>com.spotify</groupId>
-                        <artifactId>docker-maven-plugin</artifactId>
-                        <version>1.1.1</version>
+                        <groupId>org.eclipse.jkube</groupId>
+                        <artifactId>kubernetes-maven-plugin</artifactId>
+                        <version>1.18.1</version>
                         <configuration>
-                            <imageName>${docker.image.prefix}datawave/ingest-${build.env}</imageName>
-                            <forceTags>true</forceTags>
-                            <imageTags>
-                                <imageTag>latest</imageTag>
-                                <imageTag>${project.version}</imageTag>
-                            </imageTags>
-                            <dockerDirectory>${project.build.directory}/docker</dockerDirectory>
-                            <resources>
-                                <resource>
-                                    <targetPath>/</targetPath>
-                                    <directory>${project.build.directory}</directory>
-                                    <include>rpm/RPMS/noarch/${project.build.finalName}-${build.env}.rpm</include>
-                                </resource>
-                            </resources>
+                            <verbose>true</verbose>
+                            <images>
+                                <image>
+                                    <name>${docker.image.prefix}datawave/ingest-${build.env}</name>
+                                    <build>
+                                        <!-- defaults to ${project.basedir}/src/main/docker when using relative paths -->
+                                        <contextDir>.</contextDir>
+                                        <buildOptions>
+                                            <memory>8589934592</memory>
```

Comment:
Does this require the build machine have at least 8.5Gb of memory available? Should we add a note to the readme?